src/engine.h

```diff
@@ -1,4 +1,20 @@
-// Define this class here.
+//Define this class here.
+#include <stdint.h>
```

Comment:
There should be `cstdint` header file or integer definitions should work without including this. Test this out